src/Controller/Account/AccountController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the user bundle package.
+ * (c) Connect Holland.
+ */
+
+namespace ConnectHolland\UserBundle\Controller\Account;
+
+use ConnectHolland\UserBundle\Entity\User;
+use ConnectHolland\UserBundle\Entity\UserInterface;
+use ConnectHolland\UserBundle\Event\UpdateEvent;
+use ConnectHolland\UserBundle\Event\UsernameUpdatedEvent;
+use ConnectHolland\UserBundle\Form\Account\AccountType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Twig\Environment;
+
+/**
+ * @codeCoverageIgnore WIP
+ * @Route({"en": "/account", "nl": "/account"}, name="connectholland_user_account")
+ */
+final class AccountController extends AbstractController
```

Comment:
Why extend abstractController. Is those only used for retrieving the user? Because you can easily use a value resolver for this:

https://symfony.com/blog/new-in-symfony-3-2-user-value-resolver-for-controllers

Reply:
Fixed